src/qnetvo/network_ansatz.py

```diff
@@ -191,6 +189,31 @@ def ansatz_circuit(settings):
 
         return ansatz_circuit
 
+    def get_network_parameter_partitions(self):
+        """
+        A ragged array containing tuples that specify how to partition a 1D array
```

Comment:
```suggestion
        A nested list containing tuples that specify how to partition a 1D array
```
Very minor, but I found "ragged array" confusing.